build_tools/bazel/cc_binary_benchmark.bzl

```
@@ -0,0 +1,57 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+"""Creates a binary and a test for a cc benchmark target.
+
+It's good to test that benchmarks run, but it's really annoying to run a billion
+iterations of them every time you try to run tests. So we create these as
+binaries and then invoke them as tests with `--min_benchmark_time=0`.
```

nit: flag name doesn't match the code

```suggestion
binaries and then invoke them as tests with `--benchmark_min_time=0`.
```